### ISSUE TITLE
Prism syntax highlighting is active even when prism_syntax_highlighting is false

### ISSUE THREAD

**Issue by chalin** [google/docsy#729]
Visit the Docsy User Guide section [**Look and Feel** >> **Color palette and other styles**](https://www.docsy.dev/docs/adding-content/lookandfeel/#color-palette-and-other-styles). This is the first code block of that section, as syntax highlighted by Chroma:

> <img width="652" alt="Screen Shot 2021-10-16 at 3 13 19 PM" src="https://user-images.githubusercontent.com/4140793/137599501-b0baca23-cb33-476e-afcf-b82fd1e6e761.png">

You'll see that only briefly since Prism sweeps over, changing the highlighting to this:

> <img width="650" alt="Screen Shot 2021-10-16 at 3 11 09 PM" src="https://user-images.githubusercontent.com/4140793/137599448-376ba1e5-1d01-4d5f-9094-67108bc201a9.png">

The problem seems to be that MarkMap (#612) is loading prism, as we can see from the Network view:

> <img width="640" alt="Screen Shot 2021-10-16 at 3 19 41 PM" src="https://user-images.githubusercontent.com/4140793/137599682-1387f3b1-96d4-4c0e-90e3-c83ff7eaa1f7.png">

**Comment by chalin** [comment]
Actually, if you refresh a few times you might see this syntax highlighting:

> <img width="653" alt="Screen Shot 2021-10-16 at 3 29 55 PM" src="https://user-images.githubusercontent.com/4140793/137599945-79acaff3-2422-4441-a6c0-320adcd2d809.png">

This seems to correspond to when there are 404 errors in the console, like these:

```erlang
GET https://cdn.jsdelivr.net/npm/prismjs@1.23.0/components/prism-mhchem.min.js net::ERR_ABORTED 404
```

**Comment by LisaFC** [comment]
So I have "fixed" this indirectly by moving the MarkMap diagram to a different page in #750 ...

This does not solve the fact that it seems to be loading the Prism files for some reason (does the code in a MarkMap use Prism by default, I wonder?), which will affect every other code snippet on the page. I'll poke at it some more.

**Comment by LisaFC** [comment]
AHA. If I take the fenced code block out of our example MarkMap, the other code blocks on the page use Chroma, as specified. I now just need to figure out how to stop it loading Prism (or if it's possible...).

**Comment by LisaFC** [comment]
Yep, Markmap uses/loads Prism by default - you can switch it off as an option if you're running it from the command line, but we're using an autoloader script that has no parameters. Might file an issue in their repo to see if it's configurable and put a warning/tip in our docs for now - does that make sense?

**Comment by chalin** [comment]
Good job tracking that down! Sure that makes sense. If that sort of configuration isn't possible, then maybe we can simply document that hardcoded dependency in the user guide.

**Comment by chalin** [comment]
@LisaFC - any updates?

**Comment by LisaFC** [comment]
Thanks for the reminder, looks like it should now be possible to disable Prism in the autoloader so I'm going to give it a try...

**Comment by LisaFC** [comment]
Fixed in #1175
